.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,34 @@
+name: "Code scanning - action"
+
+on:
+  pull_request:
+  schedule:
+    - cron: '0 8 * * *'
```

Comment:
We generate a lot of PRs. Do we need a cron?

Reply:
I kind of agree — I think the benefit of doing this in GitHub is to run it on PRs.

Reply:
It was part of the template for this action, but yeah, I don't really see how new issues could creep in under-the-radar. I'll remove it.